cmd/tuple/import.go

```diff
@@ -32,10 +32,10 @@ import (
 )
 
 // MaxTuplesPerWrite Limit the tuples in a single batch.
-var MaxTuplesPerWrite = 1
+var MaxTuplesPerWrite = int32(1)
 
 // MaxParallelRequests Limit the parallel writes to the API.
-var MaxParallelRequests = 10
+var MaxParallelRequests = int32(10) //nolint:mnd
```

Comment:
We didn't need `nolint:mnd` previously? Do we need it on line 35 as well?

Reply:
It seems that because `10` is an argument to the casting, that's what is triggering the check here. As for line 35 `1` is by [default excluded](https://github.com/tommy-muehle/go-mnd#excludes)

Good call out though as I think I can refactor this to drop the `int32` cast which means I can drop the lint